GliaWidgets/Sources/Upload/FileUpload.Environment.Interface.swift

```diff
@@ -4,6 +4,21 @@ extension FileUpload {
     struct Environment {
         var uploadFile: UploadFile
         var uuid: () -> UUID
+
+        /// Adds ability to environment to overwrite one of its field.
```

Comment:
one of its fields

Reply:
Removed it altogether c21d0c39502db5f7d66d3fd8bed694bd10c17332.